client/src/app/+videos/+video-watch/video-watch.component.scss

```diff
@@ -3,7 +3,7 @@
 @import '_bootstrap-variables';
 @import '_miniature';
 
-$player-factor: 1.7; // 16/9
+$player-factor: 1.7777; // 16/9
```

Comment:
```suggestion
$player-factor: #{16/9}
```

Reply:
ohh, of course... I'm not used to scss and didn't know that was possible... thank you!